framework/src/application.ts

```diff
@@ -197,15 +197,15 @@ export class Application {
 		interoperabilityModule.registerInteroperableModule(tokenModule);
 		interoperabilityModule.registerInteroperableModule(feeModule);
 
-		// register modules
-		application._registerModule(feeModule); // fee should be registered first to call beforeTransactionsExecute first
+		// Register modules in the sequence defined in LIP0063 https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#modules
 		application._registerModule(authModule);
 		application._registerModule(validatorModule);
 		application._registerModule(tokenModule);
-		application._registerModule(rewardModule);
-		application._registerModule(randomModule);
-		application._registerModule(posModule);
+		application._registerModule(feeModule);
 		application._registerModule(interoperabilityModule);
+		application._registerModule(posModule);
+		application._registerModule(randomModule);
+		application._registerModule(dynamicRewardModule);
```

Comment:
Why don't we register the Legacy module? It is in the list [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#modules).

Reply:
Legacy module is developed in Lisk Core so they can inherit this sequence and simply add Legacy module as they are doing right now https://github.com/LiskHQ/lisk-core/blob/a44f9d2106f396f2a849a4d04fbbc1decf9c8573/src/application/index.ts#L23